src/main/resources/liquibase/workbench_changelog/17_4_0.xml

```diff
@@ -147,6 +147,26 @@
 			INSERT INTO role_type_permission (role_type_id, permission_id, selectable) VALUES ('3', @modify_pedigree_id, '1');
 		</sql>
 	</changeSet>
+	<changeSet author="cuenyad" id="v17.4.0-4-1">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM permission where name = 'DELETE_GERMPLASM'
+            </sqlCheck>
+        </preConditions>
+        <comment>Granular permission for delete Germplasm for Program Role Type</comment>
+		<sql dbms="mysql" splitStatements="true">
+			set @manage_germplasm_id = (select permission_id from permission where name = 'MANAGE_GERMPLASM');
+
+			insert into permission(name, description, parent_id, workbench_sidebar_category_link_id)
+			values ('DELETE_GERMPLASM', 'Delete Germplasm', @manage_germplasm_id, null);
+
+			SET @delete_germplasm_id = (SELECT permission_id FROM permission where name = 'DELETE_GERMPLASM');
+
+			INSERT INTO role_type_permission (role_type_id, permission_id, selectable) VALUES ('1', @delete_germplasm_id, '1');
+			INSERT INTO role_type_permission (role_type_id, permission_id, selectable) VALUES ('2', @delete_germplasm_id, '1');
+			INSERT INTO role_type_permission (role_type_id, permission_id, selectable) VALUES ('3', @delete_germplasm_id, '1');
```

Comment:
Can program roles delete germplasm?

Reply:
Did not get an answer from Mariano but comparing with the Edit germplasm, I assume that the same logic applies to this permission